### ISSUE TITLE
Incorrect Popover Outline

### ISSUE THREAD

**Issue by aabenoja** [WTW-IM/es-components#223]
When opening a `Popover` the popover content should have an outline to show the content has focus. At the moment the outline only appears as a dot in the top right corner of the popover. This behavior is observed in Chrome, Safari, and IE. For some reason we do not see this behavior in Firefox.
